quirks/password-rules.json

```diff
@@ -270,6 +270,9 @@
     "telekom-dienste.de": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [#$%&()*+,./<=>?@_{|}~];"
     },
+    "twitter.com": {
+        "password-rules": "minlength: 8; maxlength: 32; required: lower; required: upper; required: digit; required: [-`~!@#$%^&*()_+={}:;'\"<>,.?/|\\[]];"
```

Comment:
So it turns out that your required set is one character away from `special`, and the only character that’s missing is space (` `). Does Twitter allow spaces in passwords? If so, we should update this last part to `required: special;`.

Reply:
https://www.awesomescreenshot.com/video/487826?key=84e7ca7faf7de1c441aeac43410ff15e


Okay I have checked the condition and its working with 'space' and i have also updated the pr.